src/actions/preferences.js

```diff
@@ -13,12 +13,12 @@ import {getPreferenceKey} from 'utils/preference_utils';
 import {bindClientFunc, forceLogoutIfNecessary} from './helpers';
 import {getLogErrorAction} from './errors';
 
-export function deletePreferences(preferences) {
+export function deletePreferences(userId, preferences) {
```

Comment:
this is an example of a preference

```js
preference = {
                user_id: currentUserId,
                category: Preferences.CATEGORY_DIRECT_CHANNEL_SHOW,
                name: otherUserId,
                value: 'true'
            };
```

so if you set the prop to the userId that you want, that should be enough, right?